src/applications/edu-benefits/1990s/content/OMBInfo.jsx

```diff
@@ -96,17 +96,17 @@ class OMBInfo extends React.Component {
           </button>
```

Comment:
This button should be replaced with VaButton, but there is no variation for a button link